codes/quantum/qubits/15_qubit_code.yml

```diff
@@ -0,0 +1,91 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+# Use UTF-8 unicode encoding
+# AMS-TeX commands are rendered inside \( ... \) using MathJaX.
+# Allowed external bibliographic references are
+#    \cite{arXiv:#.#} or \cite{arXiv:quant-ph/#},
+#    \cite{doi:#}, or, as a last resort
+#    \cite{manual:{(enter citation line incl. author and year here)}}
+# External websites such as code tables, coding theory packages, github pages linked as
+#    \url{https://example.com/example}
+#    \href{https://example.com/example}{link text}
+# Internal references to codes are 
+#    \hyperref[code:code_id]{link text}
+# Delete instructional comments when submitting
+
+# code id, physical, logical are all lower case
+code_id: 15_qubit_code
+physical: qubits
+logical: qubits
+
+# Apostrophes are denoted by two apostrophe characters, i.e., ''
+# Code title is singular
+name: '[[15,1,3]] code'
+
+# First reference with the code (optional).
+#introduced: 'A 15-qubit Reed-Muller code\cite{arXiv:quant-ph/9610011}'
+
+# Anything applicable to a larger parent set of codes (see below) should go in
+# that entry instead of here.
+description: 'A 15-qubit Reed-Muller code that supports transversal \( T \) gate.\cite{arXiv:quant-ph/9610011}\cite{arXiv:1403.2734}.'
```

Comment:
Please do try to explain this code qualitatitvely. There's a nice 3D pyramid picture of it when thought of as a color code. Maybe there are explanations around in the liteature.

Reply:
Done.